src/Corrade/Utility/Test/DirectoryTest.cpp

```diff
@@ -585,12 +587,35 @@ void DirectoryTest::mkpathNoPermission() {
     if(Directory::exists("/nope"))
         CORRADE_SKIP("Can't test because the destination might be writeable");
 
+    /* Ensure errors are printed in English */
+    std::setlocale(LC_ALL, "en_US.utf8");
+
+    std::ostringstream out;
+    Error err{&out};
+
     CORRADE_VERIFY(!Directory::mkpath("/nope/never"));
+
+    CORRADE_COMPARE(out.str(),
+        "Utility::Directory::mkpath(): error: Permission denied\n"
```

Comment:
seeing this, i think it could make sense to print the actual path that failed -- which might be in 90% cases *the* most useful piece of information, the reason why it failed being secondary:

```suggestion
        "Utility::Directory::mkpath(): error creating /nope: Permission denied\n"
```

Same for the Windows case of course :)

Reply:
✅